src/frequenz/channels/_merge.py

```diff
@@ -13,13 +13,23 @@
 _T = TypeVar("_T")
 
 
-class Merge(Receiver[_T]):
-    """Merge messages coming from multiple channels into a single stream.
+def merge(*receivers: Receiver[_T]) -> Receiver[_T]:  # noqa: DOC502
+    """Merge messages coming from multiple receivers into a single stream.
+
+    Args:
+        *receivers: The receivers to merge.
+
+    Returns:
+        A receiver that merges the messages coming from multiple receivers into a
+            single stream.
```

Comment:
`The receiver` instead according to dart suggestions?

"The" also for arguments and return documentation. The rationale is that for attributes/arguments/returns you are talking about a specific instance ("this" instance), so a definite article is more appropriate, while for classes you are talking about an unspecified instance, any instance, so an indefinite article is more appropriate.

Reply:
Hahaha, well played Zullo, well played... (BTW, the rationale comes from me, not the dart docs, so you are sending me a citation of what I wrote myself :rofl: )

BTW, I thought about A/the here, and since it is not returning an attribute, but creating a complete new object, so every call will return a new one, not "the" same one, I thought "A" was more appropriate in this context. I only take these guides as suggestion, not law, and chose whatever I feel is more appropriate for each case.

Let me know if you still feel it should be updated in this case.

Reply:
haha I thought it was taken from the dart suggestions :angel: 

That's fine with me as you have thought about it and it wasn't randomly documented. I agree those are just suggestions and there is no need to be strict with them. Though I'd prefer to keep documentation consistent whenever is possible. So far, "The" seems to be always a good fit for arguments.